docs/ref/urlresolvers.txt

```diff
@@ -236,3 +236,8 @@ its web server (normally, :func:`~django.urls.reverse` takes care of this for
 you). In that case, you can call ``get_script_prefix()``, which will return
 the script prefix portion of the URL for your Django project. If your Django
 project is at the root of its web server, this is always ``"/"``.
+
+.. note::
+
+    This should not be used outside the request-response cycle since it relies
+    on values initialized during the request-response cycle.
```

Comment:
I don't like bare "this" :smile:, maybe

```suggestion
.. warning::

    This function **cannot** be used outside of the request-response cycle
    since it relies on values initialized during that cycle.
```

Reply:
Ah warning is what I was looking for! Yeah this reads better thank you :+1: